pkg/apis/kops/validation/validation.go

```diff
@@ -192,18 +192,10 @@ func validateClusterSpec(spec *kops.ClusterSpec, c *kops.Cluster, fieldPath *fie
 		}
 	}
 
-	if spec.ContainerRuntime != "" {
-		allErrs = append(allErrs, validateContainerRuntime(c, spec.ContainerRuntime, fieldPath.Child("containerRuntime"))...)
```

Comment:
We should continue to validate that this isn't set to `docker` or some other non-empty value that isn't `containerd`.